packages/eslint-plugin/configs/test-e2e-playwright.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+	extends: [ 'plugin:playwright/playwright-test' ],
```

Comment:
The documentation is missing for the new rules file. Maybe, it should be integrated directly with the top-level `.eslintrc.js` for the time being. I don't expect WordPress community projects to start using Playwright from day one. They will instead wait to see how it is used in Gutenberg and WordPress core.

Reply:
I'd prefer to place it here alongside other eslint presets, just to keep things simple. We can always add documentation later, and developers shouldn't rely on undocumented features anyway.

Reply:
What's the point to pollute the npm package with the things nobody can use but needs to install for the time being where only the Gutenberg project needs it?